docs/faq.md

```diff
@@ -3,57 +3,21 @@ title: Frequently Asked Questions
 ---
 
 ### I'm not sure what to do with Reason
-Think of what project you'd usually make if it was pure JavaScript; try porting/writing that in Reason + BuckleScript instead! We recommend trying to make concrete, end-user projects (e.g. a little command line util) rather than infra-level projects (e.g. a boilerplate generator). The latter category requires expertise and understanding idiomatic Reason code.
 
-### What's the relation between Reason, BuckleScript and OCaml?
-Reason's a syntax for OCaml and supports all its features. BuckleScript compiles OCaml/Reason code into JavaScript.
+You can do all the things you'd usually do with OCaml! OCaml is an incredible useful language for systems programming, while still being able to compile to pretty type safe JS with the aid of the `js_of_ocaml` compiler.
 
-### Where do all these `print_endline`, `string_of_int` functions come from?
-They're from the standard library, pre-`open`ed during the compilation of your file. This is why you see them in scope.
+Natively compiled CLI's are also known to be really fast (like... C-like fast), and since the language is garbage collected, you will find yourself in a very nice spot of not having to worry about borrow-checking like in Rust and you don't have to deal with verbose non-ML languages like Go. 
 
-You can read more about the Pervasives library in the api documentation:
+### What is BuckleScript and ReScript, and why is it mentioned in so many Reason related resources?
 
-https://reasonml.github.io/api/Pervasives.html
+Reason used to be tightly coupled to "BuckleScript", which used to be a OCaml-to-JS compiler like JSOO, but with different tweaks to get different JS output.
 
-### Can I have a function to print arbitrary data structures?
-If you're compiling to JavaScript through BuckleScript, you can use the JS `console.log` through [`Js.log`](https://bucklescript.github.io/bucklescript/api/Js.html#VALlog).
+On [July 1st](https://rescript-lang.org/blog/bucklescript-8-1-new-syntax), the Reason and BuckleScript leadership decided to part in different directions due to lack of common goals, and ReScript implemented its own syntax (also called ReScript) to cater a more fine-tuned experience to their JS developers. More infos on the BuckleScript -> ReScript rebranding can be found in the [official announcement](https://rescript-lang.org/blog/bucklescript-is-rebranding).
```

Comment:
@jordwalke can you take a pass at this section?

Reply:
I forgot to push some changes yesterday, but I had to revisit this part anyways.

AFAIK infos about the Reason roadmap are spread out in different github issues, PRs and Discord messages. Ideally there would be a summarized Reason blog post on the current situation of the project, so it's easier for us to cross-reference that as well.

Right now it's a little one-sided, so I created this small FAQ section to add some balance.

Reply:
@kyldvs I reduced the FAQ part to the minimum and just referred to the ReScript blog post instead. Do you think this will work without Jordan's review?